src/contracts/protocol/tokenization/RwaAToken.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.10;
+
+import {IAccessControl} from 'src/contracts/dependencies/openzeppelin/contracts/IAccessControl.sol';
+import {IncentivizedERC20} from 'src/contracts/protocol/tokenization/base/IncentivizedERC20.sol';
+import {SafeCast} from 'src/contracts/dependencies/openzeppelin/contracts/SafeCast.sol';
+import {IERC20} from 'src/contracts/dependencies/openzeppelin/contracts/IERC20.sol';
+import {Errors} from 'src/contracts/protocol/libraries/helpers/Errors.sol';
+import {AToken} from 'src/contracts/protocol/tokenization/AToken.sol';
+import {IRwaAToken} from 'src/contracts/interfaces/IRwaAToken.sol';
+import {IPool} from 'src/contracts/interfaces/IPool.sol';
+
+/**
+ * @title Aave ERC20 RWA AToken
+ * @author Aave
+ * @notice Implementation of the interest bearing token
+ * for Real-World Assets (RWAs) within the Aave protocol
+ */
+abstract contract RwaAToken is AToken, IRwaAToken {
+  using SafeCast for uint256;
+
+  /**
+   * @dev The role that can force transfer RWA aTokens
+   * This role should be granted in the ACLManager
+   */
+  bytes32 public constant ATOKEN_TRANSFER_ROLE = keccak256('ATOKEN_TRANSFER_ROLE');
```

Comment:
shouldn't we add this to the interface?

Reply:
[07fdcf5](https://github.com/aave/aave-v3-horizon/pull/1/commits/07fdcf5b78c2ef8d76b89c5c508e8a8410d6bd0c)